lusidtools/pandas_utils/lusid_pandas.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+from flatten_json import flatten
+
+
+def lusid_respose_to_pd(lusid_response):
```

Comment:
I think we need to create the `__init__.py` and export the function

Reply:
Sure, that's been added in latest commit.